components/x11/cmap_alloc/Makefile

```diff
@@ -13,45 +13,35 @@
 # Copyright 2015 Aurelien Larcher
 #
 
+BUILD_BITS=64
+BUILD_STYLE=justmake
 include ../../../make-rules/shared-macros.mk
 
-COMPONENT_NAME=           cmap_alloc
-COMPONENT_VERSION=        7.5
-COMPONENT_FMRI=           x11/cmap_alloc
+COMPONENT_NAME=       cmap_alloc
+COMPONENT_VERSION=    7.5
+COMPONENT_REVISION=   1
+COMPONENT_FMRI=       x11/cmap_alloc
 COMPONENT_CLASSIFICATION= System/X11
-COMPONENT_SUMMARY=  			cmap_alloc - colormap allocation utility
-COMPONENT_SRC=      			src
-COMPONENT_PROJECT_URL = https://hg.java.net/hg/solaris-x11~x-s12-clone
-COMPONENT_LICENSE=      Oracle
-COMPONENT_LICENSE_FILE= cmap_alloc.license
+COMPONENT_SUMMARY=    cmap_alloc - colormap allocation utility
+COMPONENT_SRC=        src
+COMPONENT_PROJECT_URL=https://hg.java.net/hg/solaris-x11~x-s12-clone
+COMPONENT_LICENSE=    Oracle
 
```

Comment:
What about "TEST_TARGET = $(NO_TESTS)" here?

Reply:
Thank you, good catch!